src/api/stats/fantom/getHectorApy.ts

```diff
@@ -0,0 +1,52 @@
+const { fantomWeb3: web3 } = require('../../../utils/web3');
+const { FANTOM_CHAIN_ID: chainId } = require('../../../constants');
+import { getEDecimals } from '../../../utils/getEDecimals';
+import { getRewardPoolApys } from '../common/getRewardPoolApys';
+
+import { addressBook } from '../../../../packages/address-book/address-book';
+const {
+  fantom: {
+    tokens: { WFTM },
+  },
+} = addressBook;
+
+const singlePool = [
+  {
+    name: 'curve-ftm-tor',
+    address: '0x24699312CB27C26Cfc669459D670559E5E44EE60',
+    rewardPool: '0x61B71689684800f73eBb67378fc2e1527fbDC3b3',
+    chainId,
+  },
+];
+
+const getHectorApy = async () => {
```

Comment:
matter of taste but import of the address book just to set 1e18 for WFTM oracle seems too much code.

it's `1e18` by default, can just drop it:

```suggestion
const getHectorApy = async () =>
  getRewardPoolApys({
    pools: singlePool,
    oracleId: 'WFTM',
    oracle: 'tokens',
    web3: web3,
    chainId,
    // log: true,
  });
```

Reply:
This worked, but only when explicitly setting `decimals: 1e18`. Turns out there is no fallback for `params.decimals`, only `pool.decimals` (see https://github.com/beefyfinance/beefy-api/blob/52c4379f91892901ee77d1eb24d86a0699b3f5be/src/api/stats/common/getRewardPoolApys.js#L52)